admin/base/views.py

```diff
@@ -5,14 +5,13 @@
 from django.utils.http import urlsafe_base64_decode
 from admin.common_auth.models import MyUser
 
-
 @login_required
 def home(request):
     context = {'user': request.user}
     return render(request, 'home.html', context)
 
 def password_reset_done(request, **kwargs):
```

Comment:
I don't think you're using this anywhere. At least not that pycharm find usages or the silver searcher could find.